### PR TITLE
fix color item height issue + refactoring

### PR DESCRIPTION
**Pull request by ronaldruzicka**
I just fixed height issue, for `.sg-color` item and refactored `.kss_example--color` row to be like bootstrap container

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -44,8 +44,27 @@ body {
...
+
+            @media (min-width: 576px) {
+                width: 540px;
+                max-width: 100%;
+            }
+            @media (min-width: 768px) {
+                width: 720px;
+                max-width: 100%;
+            }
+            @media (min-width: 992px) {
```
Comment (vbulant) [inline src/styles/main.scss]: There already is `$screen-md-min: 1024px;` on the top of the file so I’d try to reuse that var instead in order to keep the number of breakpoints as small as possible.

Then I’d add other breakpoints (576, 768, 1200) as variables as well so it’s more transparent which ones should be used.

File: src/styles/main.scss [hunk 2]
```diff
@@ -44,8 +44,27 @@ body {
...
+            padding-left: 15px;
+            padding-right: 15px;
+
+            @media (min-width: 576px) {
+                width: 540px;
+                max-width: 100%;
+            }
+            @media (min-width: 768px) {
+                width: 720px;
+                max-width: 100%;
```
Comment (vbulant) [inline src/styles/main.scss]: Duplicate of the max-width in 576px mq.

File: src/styles/main.scss [hunk 3]
```diff
@@ -44,8 +44,27 @@ body {
...
+            }
+            @media (min-width: 992px) {
+                width: 960px;
+                max-width: 100%;
```
Comment (vbulant) [inline src/styles/main.scss]: Duplicate of the max-width in 576px mq.

File: src/styles/main.scss [hunk 4]
```diff
@@ -44,8 +44,27 @@ body {
...
+            }
+            @media (min-width: 1200px) {
+                width: 1140px;
+                max-width: 100%;
```
Comment (vbulant) [inline src/styles/main.scss]: Duplicate of the max-width in 576px mq.

File: src/styles/main.scss [hunk 5]
```diff
@@ -44,8 +44,27 @@ body {
 
         &--color {
             display: flex;
-            max-width: 1140px;
+            flex-wrap: wrap;
             margin: 0 auto;
+            padding-left: 15px;
```
Comment (vbulant) [inline src/styles/main.scss]: Consider adding $gutter: 30px and reusing it like ($gutter / 2) like in bootstrap so we can reuse it.
I know, it’s like recreating bootstrap less/scss logic again but better that nothing at the moment.

**Review by vbulant — CHANGES_REQUESTED** [review]

**Review by janpanschab — APPROVED** [review]

**Pull request merged**